documentation/src/docs/getting-started.md

```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
 
 This launches all the components of Pravega on your local machine.
 
-**Note:** This is for testing/demo purposes only, *do not* use this mode of deployment 
-in Production! 
+**Note:** This is for testing/demo purposes only, **do not** use this mode of deployment
+in Production!
 
 More options to run Pravega can be found in [Running Pravega](deployment/deployment.md) guide.
-           
 
 ```
 $ cd pravega-<version>
 $ bin/pravega-standalone
 ```
 
-By using the above commands, Pravega should be up and running very soon.
+By using the above commands, Pravega should be up and running immediately.
```

Comment:
This sentence actually makes little sense. We should say:

```
The command above runs Pravega locally for development and testing purposes. It does not persist in the storage tiers like we do with a real deployment of Pravega and as such you shouldn't expect it to recover from crashes, and further, not rely on it for production use. For production use, we strongly encourage a full deployment of Pravega.
```

Reply:
Updated in the document.